src/BlazorUI/Bit.BlazorUI/Components/Navs/DropMenu/BitDropMenu.scss

```
@@ -3,35 +3,60 @@
 .bit-drm {
     width: fit-content;
-    color: $clr-fg-pri;
     font-family: $tg-font-family;
     font-weight: $tg-font-weight;
-    background-color: $clr-bg-pri;
     border-radius: $shp-border-radius;
+    color: var(--bit-drm-clr-txt);
+    background-color: var(--bit-drm-clr);
+    transition: color $mot-duration-short $mot-easing, background-color $mot-duration-short $mot-easing;
     --bit-drm-transform-factor: 1;
+    // The default look, kept identical to a BitColor of PrimaryBackground, is declared here rather than
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)